saleor/tests/e2e/checkout/taxes/test_checkout_calculate_simple_taxes_based_on_shipping_tax_class.py

```diff
@@ -18,80 +13,46 @@
 )
 
 
-def prepare_tax_configuration(
-    e2e_staff_api_client,
-    channel_slug,
-    country_code,
-    country_tax_rate,
-    shipping_tax_rate,
-    prices_entered_with_tax,
-):
-    tax_config_data = get_tax_configurations(e2e_staff_api_client)
-    channel_tax_config = tax_config_data[0]["node"]
-    assert channel_tax_config["channel"]["slug"] == channel_slug
-    tax_config_id = channel_tax_config["id"]
-
-    tax_config_data = update_tax_configuration(
-        e2e_staff_api_client,
-        tax_config_id,
-        charge_taxes=True,
-        tax_calculation_strategy="FLAT_RATES",
-        display_gross_prices=True,
-        prices_entered_with_tax=prices_entered_with_tax,
-    )
-    update_country_tax_rates(
-        e2e_staff_api_client,
-        country_code,
-        [{"rate": country_tax_rate}],
-    )
-
-    country_rates = [{"countryCode": country_code, "rate": shipping_tax_rate}]
-    tax_class_data = create_tax_class(
-        e2e_staff_api_client,
-        "Shipping tax class",
-        country_rates,
-    )
-    tax_class_id = tax_class_data["id"]
-
-    return country_tax_rate, shipping_tax_rate, tax_class_id
-
-
 @pytest.mark.e2e
 def test_checkout_calculate_simple_tax_based_on_shipping_tax_class_CORE_2009(
     e2e_staff_api_client,
     e2e_not_logged_api_client,
-    permission_manage_products,
-    permission_manage_channels,
+    shop_permissions,
     permission_manage_product_types_and_attributes,
-    permission_manage_shipping,
-    permission_manage_taxes,
 ):
     # Before
     permissions = [
-        permission_manage_products,
-        permission_manage_channels,
-        permission_manage_shipping,
+        *shop_permissions,
         permission_manage_product_types_and_attributes,
-        permission_manage_taxes,
     ]
     assign_permissions(e2e_staff_api_client, permissions)
 
-    (
-        warehouse_id,
-        channel_id,
-        channel_slug,
-        shipping_method_id,
-    ) = prepare_shop(e2e_staff_api_client)
-
-    country_tax_rate, shipping_tax_rate, tax_class_id = prepare_tax_configuration(
+    shop_data = prepare_shop(
         e2e_staff_api_client,
-        channel_slug,
-        country_code="US",
-        country_tax_rate=10,
-        shipping_tax_rate=8,
+        shipping_country_tax_rate=8,
+        billing_country_tax_rate=10,
```

Comment:
The billing country rate is not required for this test. A rename is enough at this point.
The test assumes a shipping country rate of 10% and a shipping class tax rate of 8%.